-- invalid closure call

function main() : * {
  call(2, 1)
}
